sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexerAsyncClient.java

```diff
@@ -40,6 +40,327 @@
  * This class provides a client that contains the operations for creating, getting, listing, updating, or deleting data
  * source connections, indexers, or skillsets and running or resetting indexers in an Azure Cognitive Search service.
  *
+ * <h2>
+ *     Overview
+ * </h2>
+ *
+ * <p>
+ *     Indexers provide indexing automation. An indexer connects to a data source, reads in the data, and passes it to a
+ *     search engine for indexing into a target search index. Indexers read from an external source using connection
```

Comment:
I'm not sure `search engine` is the right term here. I'd use skillset pipeline as that's what is does:

data source -> pulling data -> skillset pipeline -> index population

Reply:
I'll update the javadoc